lib/interactive_viewer.ml

```diff
@@ -4,11 +4,19 @@ open Lwd_infix
 
 let operation_info z_patches : ui Lwd.t =
   let$ z = Lwd.get z_patches in
+  let p = Zipper.get_focus z in
+  let num_hunks = List.length p.Patch.hunks in
+  let hunk_text =
+    match num_hunks with
+    | n when n > 1 || n = 0 -> Printf.sprintf "%d hunks" n
+    | _ -> "1 hunk"
```

Comment:
I just noticed that this match could be written this way:

```suggestion
    | 1 -> "1 hunk"
    | _ -> Printf.sprintf "%d hunks" n
```

Reply:
Oh wow!
This is great. Didn't know.
Thank you.

But I think `n` should be replace by `num_hunks` which is the variable that holds the hunks.
Check the attached image.
@Julow 

<img width="215" alt="image" src="https://github.com/panglesd/diffcessible/assets/126210538/0b9b0401-fb48-4fdc-ac85-eebf6b4862c7">

Reply:
You are absolutely right!

Reply:
Okay then, I have implemented the change as recommended.
Kindly review.

Thank you
@Julow